package.json

```diff
@@ -43,14 +45,22 @@
     "@ptc-org/nestjs-query-typeorm": "4.2.1-alpha.2",
     "@react-email/components": "0.0.12",
     "@react-email/render": "0.0.10",
+    "@react-pdf/renderer": "^4.1.6",
     "@sentry/node": "^8",
     "@sentry/profiling-node": "^8",
     "@sentry/react": "^8",
     "@sniptt/guards": "^0.2.0",
     "@stoplight/elements": "^8.0.5",
     "@swc/jest": "^0.2.29",
     "@tabler/icons-react": "^2.44.0",
-    "@tiptap/extension-hard-break": "^2.9.1",
+    "@tiptap/core": "^2.10.4",
```

Comment:
We're actually trying to move away from the centralized package.json whenever we can (we went in that direction initially but it mostly had bad consequences so now we're moving the other way around. Unless it's really a package that's shared across backend and frontend). Do you think you could keep this in the twenty-front package.json? I don't see why it would break

Reply:
Thank you for the clarification! I now understand the rationale behind moving away from the centralized package.json. The packages I moved to the root package.json were not causing any breakage when defined in twenty-front/package.json. My intention was to align with what appeared to be the existing pattern, as I noticed many packages were already defined at the root level. Given your feedback, I'll revert this change and ensure the packages remain in twenty-front/package.json.

Reply:
Resolved in recent commit.